packages/app/public/favicon.svg

```diff
@@ -0,0 +1,17 @@
+<svg id="Assets" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 337.46 428.5">
+    <title>
+        favicon
+    </title>
+    <style>
+        path {
```

Comment:
This is going to affect **all** `path` elements in the document, we don't want that. Instead, use `withStyles` from the `@material-ui/core` and also then you are able to read `theme.palette.type` which would be `light` or `dark` based on the current theme used.

Reply:
Sorry, misread the context, disregard the material-ui part. Anyway we need to make this more modular, at least like a className